src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  font-weight: 350;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-style: bold;
+  font-weight: 850;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin-top: 0;
+  padding-top: 20px;
+  font-family: Avenir, sans-serif;
+}
+
+.form {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.label {
+  padding: 0;
+  margin: 0;
+  border: none;
+}
+
+.input {
+  width: 100%;
+  margin-top: 0;
+  margin-bottom: 20px;
+  height: 70px;
+  padding-top: 25px;
+  padding-bottom: 23px;
+  padding-left: 62px;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 100%;
+  color: #3d4e61;
+  font-style: normal;
+  font-weight: 350;
+  background: #fff url(./images/Search.svg) no-repeat 26px;
+  background-size: 19px 19px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.inputSmall {
```

Comment:
Here you as well has a lot of similar styles. Move it under one rule set, then specify only different ones